### ISSUE TITLE
Bug: Crossbow provides suggestions for Metadata

### ISSUE THREAD

**Issue by kraibse** [shoedler/crossbow#6]
Hello again,

I noticed there are empty suggestions that are referring to the space before the metadata separators ( --- ).

![grafik](https://user-images.githubusercontent.com/33869366/234855072-fd01a130-2828-45ce-974d-a12778a65fd2.png)

I checked the suggested heading and it turned out that it has no text and only consists of the ( # ) symbol.

**Comment by shoedler** [comment]
Thank you for pointing this out! I'll update this issue once I get around to fixing it

**Comment by shoedler** [comment]
I'm unable to reproduce this, could you give me some guidance on how you got these empty suggestions?

**Comment by kraibse** [comment]
Sure, what I did to reproduce this was as follows:

- Created a reference file with an empty heading ( `## ` or `##` both activate it)
- Created a new file with a metadata attribute, e.g. date of creation

Contents of file1.md
```
##

This file causes the empty suggestion.
```

Contents of file2.md
```
---
created: 2023-05-06
---
Here the metadata tags will trigger the empty suggestion before the tokens `---`.
```

After applying the suggestion to file2.md:
```
[[file1.md#]]---
created
---
```

**Comment by shoedler** [comment]
I've just released [1.2.1](https://github.com/shoedler/crossbow/releases/tag/1.2.1), this should be fixed. 
Please confirm and close this issue. If you still encounter the same bug, notify me on this channel.

**Comment by shoedler** [comment]
Closing as fixed, reopen if you still encounter this issue
